### ISSUE TITLE
Toolbar / top menu missing when "Request Course"

### ISSUE THREAD

**Issue by leonardmensah** [open-learning-exchange/BeLL-Apps#160]

**Comment by SabaBaig** [comment]
Actually it's the default behavior (part of version 0.11.69) of BeLL-Apps. Also, it does not fulfill the business logic. Since, if someone has already clicked on "Request Resources", then it is for sure that he will be requesting resource(s) rather than navigating.

**Comment by leonardmensah** [comment]
Exactly.. Thus that mean when I mistakenly click on request resource, I should be deprived of navigating to other menus ?. No. I don't think so
Users should have access to basic navigation options like the top menu

**Comment by SabaBaig** [comment]
This action can be reverted by pressing the cancel button, which is there on the form.

**Comment by leonardmensah** [comment]
Which action.. The menu not appearing on the "Request Course " or when users click on back instead of Cancel ?
